shared/actions/chat/index.js

```diff
@@ -778,7 +778,13 @@ function* _startConversation(action: Constants.StartConversation): SagaGenerator
   }
 
   const inboxSelector = (state: TypedState, tlfName: string) => {
-    return state.chat.get('inbox').find(convo => convo.get('participants').sort().join(',') === tlfName)
+    return state.chat
+      .get('inbox')
+      .find(
+        convo =>
+          convo.get('participants').sort().join(',') === tlfName &&
+          convo.get('membersType') === ChatTypes.CommonConversationMembersType.kbfs
```

Comment:
not that the perf matters so much but i'd put this one first since its quicker than doing a sort/join